ui/src/translations/fr.json

```diff
@@ -135,5 +136,10 @@
   "please_refer_to": "Veuillez vous référer à",
   "import_solution_hint": "Vous devez importer une solution pour préparer votre environnement",
   "redirecting": "Redirection…",
-  "documentation": "Documentation"
+  "documentation": "Documentation",
+  "env_preparation": "Environment preparation",
+  "env_preparation_success": "L'environnement {envName} est en cours de préparation.",
+  "env_preparation_failed": "La préparation de l'environnement {envName} a échoué.",
+  "preparing_environemnt": "préparer l'environnement",
```

Comment:
This one is a bit weird too, what's the use/context?

If it's for a progress bar or some kind of load, then `Préparation de l’environnement…` would be better (and English could be `Preparing the environment…`)

Reply:
Thanks for the French speaker^^
The context of `preparing_environemnt` is that we want to display a text next to the loader when we are waiting for preparing the environment.